adam-hbase/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,18 @@
+# Root logger option
+log4j.rootLogger=INFO, stderr, logfile
```

Comment:
Do we need this in addition to the adam-core log4j property file? As an aside, I think that @ryan-williams was making a compelling argument recently for not committing log4j property files, but I can't quite remember it.